apps/mobile/src/components/Notification/Notifications/GalleryAnnouncement.tsx

```diff
@@ -0,0 +1,107 @@
+import clsx from 'clsx';
+import { useCallback } from 'react';
+import { Linking, View } from 'react-native';
+import FastImage from 'react-native-fast-image';
+import { graphql, useFragment } from 'react-relay';
+
+import { Button } from '~/components/Button';
+import { Typography } from '~/components/Typography';
+import { GalleryAnnouncementFragment$key } from '~/generated/GalleryAnnouncementFragment.graphql';
+import { GalleryAnnouncementQueryFragment$key } from '~/generated/GalleryAnnouncementQueryFragment.graphql';
+import { contexts } from '~/shared/analytics/constants';
+
+import { NotificationSkeleton } from '../NotificationSkeleton';
+
+type Props = {
+  notificationRef: GalleryAnnouncementFragment$key;
+  queryRef: GalleryAnnouncementQueryFragment$key;
+};
+
+export function GalleryAnnouncement({ queryRef, notificationRef }: Props) {
+  const notification = useFragment(
+    graphql`
+      fragment GalleryAnnouncementFragment on GalleryAnnouncementNotification {
+        __typename
+        title
+        description
+        ctaText
+        ctaLink
+        imageUrl
+        ...NotificationSkeletonFragment
+      }
+    `,
+    notificationRef
+  );
+
+  const query = useFragment(
+    graphql`
+      fragment GalleryAnnouncementQueryFragment on Query {
+        ...NotificationSkeletonQueryFragment
+      }
+    `,
+    queryRef
+  );
+
+  const { title, description, ctaText, ctaLink, imageUrl } = notification;
+
+  const handlePress = useCallback(() => {
+    if (!ctaLink) return;
+    Linking.openURL(ctaLink);
```

Comment:
only small thing is `ctaLink` might be a relative link, like `/jakz`, in which case we'd want to deep link to their profile, with a slightly modified version of `handleDeepLinkNavigation`

for now though we can assume `ctaLink` will always be an external link

Reply:
Sounds good! Will add deep link support in future. I'll create the ticket to keep track this

Reply:
legend